modules/UI/videolayout/Filmstrip.js

```diff
@@ -433,11 +423,14 @@ const Filmstrip = {
             promises.push(new Promise((resolve) => {
                 // Let CSS take care of height in vertical filmstrip mode.
                 if (interfaceConfig.VERTICAL_FILMSTRIP) {
-                    resolve();
+                    $('#filmstripLocalVideo').animate({
+                        // adds 4 px because of small video 2px border
+                        width: local.thumbWidth + 4
```

Comment:
Adding 4px here is quite shady, but we were already doing it below for the horizontal filmstrip, so I decided it's not that bad. The below code already got out of sync with the css though, so maybe getting the border from the component will be better.